shared/js/ui/views/enable-prompt.es6.js

```diff
@@ -0,0 +1,44 @@
+const Parent = window.DDG.base.View
+
+function EnablePrompt (ops) {
+    this.template = ops.template
+    this.model = ops.model
+    this.siteView = ops.siteView
+    this.$body = window.$('body')
+
+    Parent.call(this, ops)
+
+    this._setup()
+}
+
+EnablePrompt.prototype = window.$.extend({},
+    Parent.prototype,
+    {
+        _setup: function () {
+            this._cacheElems('.js-enable-prompt', [
+                'activate',
+                'content',
+                'success'
+            ])
+
+            this.bindEvents([
+                [this.$activate, 'click', this._onActivateClick]
+            ])
+
+            this.$body.addClass('blocking-disabled')
```

Comment:
We add this class here in order to hide the 'report broken site' option in the hamburger menu when tracker blocking is disabled.